Fix `sync.py` for `lock.json` existing but empty
